tools/LinuxOneShot/SetupProgram/PreCompile.sh

```diff
@@ -66,7 +66,7 @@ cd ..
 echo "Compiling tgui..."
 cd "$1"
 chmod +x tools/bootstrap/node  # Workaround for https://github.com/tgstation/tgstation-server/issues/1167
-env TG_BOOTSTRAP_CACHE="$original_dir" TG_BOOTSTRAP_NODE_LINUX=1 tools/bootstrap/node tools/build/build.js tgs
+env TG_BOOTSTRAP_CACHE="$original_dir" TG_BOOTSTRAP_NODE_LINUX=1 CBT_BUILD_MODE="TGS" tools/bootstrap/node tools/build/build.js
```

Comment:
Uhmm, why are there two of this line shown here. GitHub being wack

Reply:
i undid the changes to precompile here